default.json

```diff
@@ -28,7 +28,8 @@
   "prConcurrentLimit": 0,
   "branchConcurrentLimit": 0,
   "dependencyDashboard": false,
-  "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
+  "gitAuthor": "github-actions[bot] <github-actions[bot]@users.noreply.github.com>",


```

Comment:
```suggestion
  "gitAuthor": "github-actions[bot] <github-actions[bot]@users.noreply.github.com>",
```

Reply:
Not sure how the github web editor managed to add this, bust be a bug 😡